src/main/java/io/github/redstoneparadox/oaktree/test/Tests.java

```diff
@@ -7,8 +7,10 @@
 import io.github.redstoneparadox.oaktree.networking.OakTreeNetworking;
```

Comment:
I'm going to be moving the test classes into their own module before the next release and get everything reorganized, so yeah.

Reply:
Modules kinda suck. Please reconsider.

Reply:
Sorry, I meant a source set, not a module.